catalog-info.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: golden-path-nodejs
+  annotations:
+    github.com/project-slug: hmcts/golden-path-nodejs
+spec:
+  type: other
```

I think this should be documentation
```suggestion
  type: documentation
```